### DISCUSSION TITLE
ImportError: No module named rtmidi

### DISCUSSION THREAD

**Discussion by samster395** [mido/mido#434] [Q&A]
Would have used the google group but it says it doesn't exist or I don't have permission.

pip3 install python-rtmidi returns: Requirement already satisfied: python-rtmidi

**Comment by bast** [comment]
Is the problem still there? If yes, you can also ask on "Discussion".

**Comment by Liujingxiu23** [comment]
Yes, problem still exist, and when run  “pip install python-rtmidi “， it can not be installed successfully

**Comment by rdoursenaud** [comment]
Please have a look at all the other discussions having the same problem and see if their answers help you.

Otherwise, please post the errors you get upon trying to install `python-rtmidi`.
